src/main/java/org/isf/examination/dto/PatientExaminationDTO.java

```diff
@@ -0,0 +1,185 @@
+package org.isf.examination.dto;
+
+import io.swagger.annotations.ApiModelProperty;
+import org.isf.patient.dto.PatientDTO;
+
+import java.sql.Date;
+
+public class PatientExaminationDTO {
+
+    @ApiModelProperty(notes = "Patient Examination Id", example = "1", position = 1)
+    private int pex_ID;
+
+    @ApiModelProperty(notes = "Date of Patient Examination", example = "2020-03-19T14:58:00.000Z", position = 2)
+    private Date pex_date;
+
+    @ApiModelProperty(notes = "Patient Examination", position = 3)
+    private PatientDTO patient;
+
+    @ApiModelProperty(notes = "Patient Height", position = 4)
+    private Integer pex_height;
+
+    @ApiModelProperty(notes = "Patient Weight", position = 5)
+    private Double pex_weight;
+
+    private Integer pex_ap_min;
+
+    private Integer pex_ap_max;
+
+    private Integer pex_hr;
+
+    private Double pex_temp;
+
+    private Double pex_sat;
+
+    private Integer pex_rr;
+
+    private String pex_auscultation;
+
+    private Integer pex_hgt;
+
+    private Integer pex_diuresis;
+
+    private String pex_diuresis_desc;
+
+    private String pex_bowel_desc;
+
+    private String pex_note;
+
```

Comment:
Is there a particular reason for not annotating the other properties?

Reply:
You are right, I didn't know the meaning of all properties :-)

Reply:
Added with https://github.com/informatici/openhospital-api/pull/59/commits/56ca102a38431292987512aa5767ce773a5df91e